skimage/exposure/exposure.py

```diff
@@ -612,8 +612,8 @@ def _adjust_gamma_u8(image, gamma, gain):
     """LUT based implementation of gamma adjustment.
 
     """
-    lut = (255 * gain * (np.linspace(0, 1, 256) ** gamma))
-    lut = np.minimum(lut, 255).astype('uint8')
+    lut = 255 * gain * (np.linspace(0, 1, 256) ** gamma)
+    lut = np.minimum(lut + 0.5, 255).astype('uint8')
```

Comment:
```suggestion
    # Add 0.2 to ensure that round near integer boundaries are more predictable
    lut = np.minimum(lut + 0.2, 255).astype('uint8')
```
This needs a comment in the code, but I'm not sure that 0.5 is the right choice.

I feel like it will introduce an other bug somewhere. For example when the user halves things.

Reply:
Thank you @hmaarrfk for your review.
For rounding floating point to the closest integer value, adding 0.5 before casting is a very well known approach (see for example [wikipedia](https://en.wikipedia.org/wiki/Rounding#Rounding_to_the_nearest_integer) ).

The strategy I proposed corresponds to the *Round half up* in wikipedia article. We may also choose to use `np.round` instead, which corresponds to the *Round half to even* strategy.
The only difference will be when the fraction part of a value in `lut` is exactly 0.5 (which I think is really rare), and this is a matter of choice. One or the other strategy are valid :slightly_smiling_face:

Reply:
Thanks for your PR @rfezzani! Why not use [`np.rint`](https://numpy.org/doc/stable/reference/generated/numpy.rint.html) which is also faster than `np.round`?

Reply:
Any of `np.rint`, `np.round`, `np.fix` or adding 0.5 before casting are valid options here... Timing gives

```python
In [21]: gamma = 0.5

In [22]: gain = 1

In [23]: lut = 255 * gain * (np.linspace(0, 1, 256) ** gamma)

In [24]: %timeit np.round(lut).astype('uint8')
3.52 µs ± 23 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [25]: %timeit np.minimum(np.around(lut), 255).astype('uint8')
4.14 µs ± 7.63 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [26]: %timeit np.minimum(np.fix(lut), 255).astype('uint8')
7.9 µs ± 82.8 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [27]: %timeit np.minimum(np.rint(lut), 255).astype('uint8')
2.69 µs ± 13.5 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [28]: %timeit np.minimum(lut + 0.5, 255).astype('uint8')
3.11 µs ± 19 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
```
`np.rint` is in fact really efficient, it is also more explicit (no more need for a comment), I will update the PR :wink: Thank you @aqc-carlodri